test/index.ts

```diff
@@ -6,17 +6,22 @@ import {
   deployments as hardhatDeployments,
   network,
   ethers as hardhatEthers,
+  getNamedAccounts,
 } from "hardhat";
 import{
   Aludel,
   AludelFactory,
+  AludelV2,
```

Comment:
if the idea is to only use AludelV2, keep in mind the old Aludel is still being used for the one launched by the factory, line 98